utilities/cwp/custom-war-packager.inc

```diff
@@ -0,0 +1,90 @@
+# function to download the CWP jar
+# $1: Working directory
+# $2: CWP version to use
+download_cwp() {
+
+	if [ "$#" -gt 1 ]
+	then
+		cwp_jar_file="$1/.cwp-build/cwp-cli-$2.jar"
+		if [ "$#" -eq 4 ]
+		then
+            image_tag="$4"
+        else
+            image_tag="jenkins-experimental/jenkinsfile-runner-test"
+        fi
+
+		rm -rf "$1/.cwp-build"
+		mkdir -p "$1/.cwp-build"
+		wget -O "$cwp_jar_file" https://repo.jenkins-ci.org/releases/io/jenkins/tools/custom-war-packager/custom-war-packager-cli/$2/custom-war-packager-cli-$2-jar-with-dependencies.jar
+
+		echo "CWP downloaded as $cwp_jar_file"
+
+		return 0
+	else
+		echo "Error. Missing parameters:"
+		echo "   Working directory"
+		echo "   CWP version to use"
+		return 1
+	fi
+
+}
+
+# function to generate the docker image using CWP using a downloaded jar
+# $1: Working directory
+# $2: Path to cwp.jar
+# $3: Jenkins version
+# $4: Path to packager config file
+# $5: Tag for the docker image. "jenkins-experimental/jenkinsfile-runner-test" by default
```

Comment:
I would default to: "jenkinsfile-runner-test/${TEST_NAME}" if possible

Reply:
Yes, I agree.

Reply:
Not sure if possible, but I'll have a look to see if it's feasible.

Reply:
It's done in the last commit